spec/requests/users_spec.rb

```diff
@@ -3,11 +3,37 @@
 RSpec.describe "Users", type: :request do
 
   fixtures :users, :contacts, :phones
-  
+
   describe "GET /index" do
     it "should returns http success" do
-      get "/users"
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it "should render the index template" do
+      get users_path
+      expect(response).to render_template("index")
+    end
+  end
+
+  describe "GET /show" do
+    it "should returns http success" do
+      get user_path(User.find_by(id: 1))
```

Comment:
Quando for buscar a instância de um objeto pelo id você pode usar o método `find`. O `find_by` é usado quando você precisa buscar através de algum atributo que não seja o `id`